project/Dependencies.scala

```diff
@@ -58,7 +58,7 @@ object Dependencies {
       "io.reactivex" % "rxjava-reactive-streams" % "1.2.1", //ApacheV2
       "org.reactivestreams" % "reactive-streams" % "1.0.2", //CC0	http://creativecommons.org/publicdomain/zero/1.0/
       "com.typesafe.play" %% "play-json" % "2.6.9" % Test, // MIT like: http://www.slf4j.org/license.html
-      "com.typesafe.akka" %% "akka-http" % "10.1.4" % Test, // MIT like: http://www.slf4j.org/license.html
+      "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion % Test, // MIT like: http://www.slf4j.org/license.html
```

Comment:
The license comment seems to have been copy and wasted. I wonder if these comments are useful at all - now that we show licenses in the dependencies listing in the docs.

Reply:
Good point. If we ever add building documentation from PRs, then it will be easy to see the licenses of the changed dependencies or new connectors. Then we will be able to remove these comments from build sources.